src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -38,7 +50,7 @@
     <None Include="project.json" />
     <None Include="windows\project.json" />
```

Comment:
@dagood -- could you elaborate on the windows\project.json file addition?  Thanks.